styleguide/components/41-organisms/feedback-form/feedback-form.twig

```diff
@@ -0,0 +1,39 @@
+<div class="feedback-form {{ modifier }}">
+  <h3><button>{{ 'Can we improve something about this page?' }}<i class="icon-chevron-down" aria-hidden="true"></i></button></h3>
```

Comment:
Heading level seems incorrect.
It's feedback on the page, so the heading level should be just beneath the page heading level.

Now you only provide feedback on the last h2 section.

Reply:
This is designed to be a h3. So we could change it to an h2 and give it a styling to look like an h3 or we could just use the h2 but than the design will be inconsistent with the implementation.

Reply:
I'm don't like designing headers to look like other levels.
WCAG proposes consistent components to help users navigate your website. 

Either way this should be a UX decision. 
Make it a h2 with h3 styling for now and I'll raise some awareness regarding consistent header styling in the meanwhile.

Reply:
It's changed. Now go raise some awareness! :)